lld/test/ELF/lto/armeb.ll

```diff
@@ -0,0 +1,12 @@
+; REQUIRES: arm
```

Comment:
See the tests I added in abd80ecf6ed8f013b8c78fb00a2637dfc81e50cb . The comment convention is like `;; Test we can infer the e_machine value EM_RISCV from a bitcode file.`

(Double `;` to make comments oustanding from RUN/CHECK)

thumb needs to be tested as well.

Reply:
updated, thanks for your review : )

Reply:
You can use `arm.ll`. I added some tests there.

Reply:
@MaskRay thanks, I updated it accordingly.